src/components/LIcon.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div>
+        <slot></slot>
+    </div>
+</template>
+
+<script>
+import findRealParent from '../utils/findRealParent.js';
+import { optionsMerger } from '../utils/optionsUtils.js';
+import Options from '../mixins/Options.js';
+
+export default {
+  name: 'LIcon',
+  mixins: [Options],
+  props: {
+    iconUrl: String,
```

Comment:
On the props please define the default value even if `null` or `undefined` this triggers linting errors :)